packages/browser-wallet/src/popup/state/index.ts

```diff
@@ -10,3 +10,4 @@ export const addToastAtom = atom<null, string>(null, (get, set, newToast) => {
     const updatedToasts = [...currentToasts, newToast];
     set(toastsAtom, updatedToasts);
 });
+export const creatingCredentialRequestAtom = atom<boolean>(false);
```

Comment:
This has to be saved in session storage, otherwise I can just close the popup and open it again and then get access to creating duplicate accounts.